import-export-cli/cmd/set.go

```diff
@@ -144,17 +122,11 @@ func init() {
 		defaultExportDirectory = mainConfig.Config.ExportDirectory
 	}
 
-	if mainConfig.Config.TokenType != "" {
-		defaultTokenType = mainConfig.Config.TokenType
-	}
-
 	SetCmd.Flags().IntVar(&flagHttpRequestTimeout, "http-request-timeout", defaultHttpRequestTimeout,
 		"Timeout for HTTP Client")
 	SetCmd.Flags().StringVar(&flagExportDirectory, "export-directory", defaultExportDirectory,
 		"Path to directory where APIs should be saved")
-	SetCmd.Flags().StringVarP(&flagTokenType, "token-type", "t", defaultTokenType,
-		"Type of the token to be generated")
-	SetCmd.Flags().StringVarP(&flagKubernetesMode, "mode", "m", utils.DefaultEnvironmentName, "If mode is set to \"k8s\", apictl " +
-		"is capable of executing Kubectl commands. For example \"apictl get pods\" -> \"kubectl get pods\". To go back " +
+	SetCmd.Flags().StringVarP(&flagKubernetesMode, "mode", "m", utils.DefaultEnvironmentName, "If mode is set to \"k8s\", apictl "+
```

Comment:
Shall we add a space between the "(Quote symbol) and "+" at the end of the line?

Reply:
Fixed with f75a26712774c479a96a139dc5fdc95d089bb104